src/beautifiers/prettydiff.coffee

```diff
@@ -130,3 +131,23 @@ module.exports = class PrettyDiff extends Beautifier
       resolve(result)
 
     )
+
+  # Retrieves the default line ending based upon the Atom configuration
+  #  `line-ending-selector.defaultLineEnding`. If the Atom configuration
+  #  indicates "OS Default", the `process.platform` is queried, returning
+  #  CRLF for Windows systems and LF for all other systems.
+  # Code modified from atom/line-ending-selector
+  # returns: The correct line-ending character sequence based upon the Atom
+  #  configuration, or `null` if the Atom line ending configuration was not
+  #  recognized.
+  # see: https://github.com/atom/line-ending-selector/blob/master/lib/main.js
+  getDefaultLineEnding= ->
```

Comment:
- [ ] Can you move this into `beautifier.coffee` along with https://github.com/Glavin001/atom-beautify/blob/c3e554d9c48ac6880bed8bd9bf3bd8407b855438/src/beautifiers/js-beautify.coffee#L62-L80 ?
Since they are used in multiple places it should be centralized within the `beautifier` class.